src/components/modal/login.tsx

```diff
@@ -0,0 +1,37 @@
+import './login.scss';
+import { useState } from 'react';
+
+
+const SignInModal = () => {
+    const [className, setClassName] = useState(false)
+    const handleBlur = (e: React.ChangeEvent<HTMLInputElement>) => {
+        e.target.value ? setClassName(true) : setClassName(false)
+    }        
+    
+    return (
+        <div className='signInWrapper w-100' onClick={(e) => {e.stopPropagation()}}>
+            <div className='signInCredentials'>
+                <h3>Sign in</h3>
+                <hr />                
+                <div className='input-container'>
+                    <input type='text' id='username' className={`p-1 w-100 ${className ? 'is-valid' : ''}`} 
+                    onBlur={(e) => handleBlur(e)}></input>
+                    <label htmlFor='username' className=''>Username</label>
```

Comment:
What if we want to run validation checks? @onyi2020

Reply:
Whatkind of validation checks?

Reply:
I just meant in the future, if we want to check against what a user types @onyi2020

Reply:
Maybe we can do it inside the handleBlur function or when submt is triggered

Reply:
ohk @onyi2020